apps/formbricks-com/pages/docs/getting-started/quickstart-in-app-survey/index.mdx

```diff
@@ -0,0 +1,136 @@
+import { Layout } from "@/components/docs/Layout";
+import Image from "next/image";
+
+import I1 from "./1-in app surve or popup survey setup.webp";
+import I2 from "./2-settings for survey popup in app for feedback.webp";
+import I3 from "./3-web app survey settings for in app survey popup.webp";
+import I4 from "./4-in app survey trigger for popup survey.webp";
+import I5 from "./5-options survez popup in app for feedback.webp";
+import I6 from "./6-setup in app survey popup feedback box.webp";
+import I7 from "./7-in app survey popup for feedback.webp";
+import I8 from "./8-pop up form in web app survey.webp";
+import I9 from "./9-set up in app micro survey popup.webp";
+import I10 from "./10-micro survey pop up in app.webp";
+import ReactApp from "../react/1-in app survey react app popup form.webp";
+
+export const meta = {
+  title: "Collect in app survey responses in 10 minutes",
+  description: "Get your first in app survey response in 10 minutes.",
+};
+
+In app surveys have 6-10x better conversion rates then emailed out surveys. This tutorial explains how to run an in app survey in your web app in 10 to 15 minutes. Let’s go!
```

Comment:
`In app` -> `In-App` ?
`then` -> `than`

Reply:
"in app survey" is a kw with high search volume :)  A hyphen changes it